Source/Core/Core/HW/HSP/HSP_Device.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <memory>
+#include "Common/CommonTypes.h"
+
+class PointerWrap;
+
+namespace HSP
+{
+enum class HSPDeviceType : int
+{
+  HSPDEVICE_NONE,
+  HSPDEVICE_ARAM_EXPANSION,
+};
```

Comment:
Since this is an enum class and thus these values don't leak into the parent `HSP` namespace, they don't need to be prefixed with `HSPDEVICE_` anymore (and could be changed from all capse, so `None` and `ARAMExpansion`).